include/nighthawk/adaptive_load/metrics_plugin.h

```diff
@@ -11,6 +11,14 @@
 
 namespace Nighthawk {
 
+// Describes the period in which a Metrics plugin should measure.
+struct MeasuringPeriod {
+  // The beginning of the time period where nighthawk is measuring.
+  google::protobuf::Timestamp start_time;
+  // The duration of the time period nighthawk is measuring.
```

Comment:
Similarly, let's expand this comment too.

Reply:
Done.